packages/components/src/Actions/Action/Action.component.js

```diff
@@ -121,6 +121,7 @@ Action.propTypes = {
 	label: PropTypes.string.isRequired,
 	link: PropTypes.bool,
 	model: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+	name: PropTypes.string,
```

Comment:
why this is needed ?
For me a proptypes is needed only if used in the render.
if the props is passed by spread it should not be here.